atom/browser/api/atom_api_session.cc

```diff
@@ -429,12 +424,13 @@ void Session::DoCacheAction(const net::CompletionCallback& callback) {
                      action, callback));
 }
 
-void Session::ClearStorageData(mate::Arguments* args) {
-  // clearStorageData([options, callback])
+v8::Local<v8::Promise> Session::ClearStorageData(mate::Arguments* args) {
+  v8::Isolate* isolate = v8::Isolate::GetCurrent();
+  util::Promise promise(isolate);
```

Comment:
Prefer getting the isolate from simple getters when available, for instance here it can be from `args->isolate()` rather than using `v8::Isolate::GetCurrent()` which tries to obtain it from a TLS.